images/kubemci/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM gcr.io/k8s-testimages/kubekins-e2e:v20180205-d99d9c246-master
+LABEL maintainer "nikhiljindal@google.com"
+
+RUN apt-get update && apt-get install -y \
+    git && \
+    apt-get clean
+
+
+# Build latest kubemci from HEAD.
+RUN cd /tmp && \
+    git clone https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress.git && \
+    ( make build && \
```

Comment:
what's the point of the parens? It doesn't seem like you are ensuring that "rm -rf" is run.